CONTRIBUTING.md

```diff
@@ -1,290 +1,33 @@
 # Contributing
 
-Thanks for your help! Due to ReScript's nature, the contribution setup isn't all straightforward. If something isn't working, please file an issue!
+Thanks for your help! Melange's development setup is straightforward, but if something isn't working, please file an issue.
```

Comment:
This intro copy is just a remix of what was there previously, so it might be nice to write something more meaningful...not sure exactly what that would be though 😅